packages/common/src/store/account/sagas.ts

```diff
@@ -242,7 +242,14 @@ export function* fetchAccountAsync({
 
   if (user.handle) {
     // guest account don't have handles
-    yield* call(recordIPIfNotRecent, user.handle)
+    try {
+      yield* call(recordIPIfNotRecent, user.handle)
```

Comment:
why might this fail?

Reply:
recordIP request requires that blockchainUserId exists in identity for that user, which requires that authMiddleware has sent a request to discovery to fill that in, which requires that the selected DN returns back that user. That's a fair number of contingent things that need to happen, and something as little as a network hiccup can interrupt. But it _should_ be more resilient after this PR, given the retry logic baked in SDK! I added the try catch because this doesn't seem like something we want to break the login/account fetch flow regardless of how successful it is on average